db/migrate/20160205141721_remove_newest_redirects_table.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveNewestRedirectsTable < ActiveRecord::Migration
+  def up
+    drop_table :newest_redirects
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration, "newest_redirect table can't be recreated"
```

Comment:
I would like to know if this is standard way of doing this kind of migrations here.
I'm happy to change to whatever it should be.